packaging/cloudify-rabbitmq.spec

```diff
@@ -0,0 +1,46 @@
+
+Name:           cloudify-rabbitmq
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+BuildArch:      noarch
+Summary:        Cloudify's RabbitMQ configuration
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-amqp-influxdb
```

Comment:
@funkyHat That's probably not the right URL. I wonder if it's the same in the other specs?

Reply:
whoops https://github.com/cloudify-cosmo/cloudify-manager/pull/1135